psi4/driver/procrouting/proc.py

```diff
@@ -2441,7 +2442,7 @@ def run_dfmp2_gradient(name, **kwargs):
         ['DF_BASIS_SCF'],
         ['DF_BASIS_MP2'],
         ['SCF_TYPE'])
-
+    
```

Comment:
Run clang-format

Reply:
there shouldn't be any changes in proc.py at all :(